pkg/platform/model/svc.go

```diff
@@ -176,6 +177,29 @@ func (m *SvcModel) GetProcessesInUse(ctx context.Context, execDir string) ([]*gr
 	return response.Processes, nil
 }
 
+// GetJWT grabs the JWT from the svc, if it exists.
+// Note we respond with mono_models.JWT here for compatibility and to minimize the changeset at time of implementation.
+// We can revisit this in the future.
+func (m *SvcModel) GetJWT(ctx context.Context) (*mono_models.JWT, error) {
+	logging.Debug("Checking for GetJWT")
+	defer profile.Measure("svc:GetJWT", time.Now())
+
+	r := request.NewJWTRequest()
+	resp := graph.GetJWTResponse{}
+	if err := m.request(ctx, r, &resp); err != nil {
+		return nil, errs.Wrap(err, "Error sending messages request")
+	}
+
+	jwt := mono_models.JWT{}
+	err := json.Unmarshal(resp.Payload, &jwt)
+	if err != nil {
+		return nil, errs.Wrap(err, "Error unmarshaling JWT")
+	}
+	logging.Debug("Response: %#v\n", jwt)
```

Comment:
Hmm, I've never seen this kind of format parameter before. They're getting more and more complex...

Reply:
I should've dropped that, that was me debugging something.

Fwiw this variant of `%v` gives you more information about the structure, and not just its values.

https://gobyexample.com/string-formatting